packages/core/src/components/ChoiceAnswer.less

```diff
@@ -6,4 +6,10 @@
     padding: 0;
     margin: 0.25rem 0;
   }
+
+  .e-correct-answer {
+    border-left: .5rem solid #5acd46;
```

Comment:
Colors should used via colors.less (i.e. `@color[foobar]` instead of hard-coded value)